### PR TITLE
Work from 05/25 and 05/26

### PR DESCRIPTION
**Pull request by nif33**
Progress made from yesterday and today on Project 1: front end website using HTML, CSS, and Javascript

(Sorry that it's a lot of changes, I did not have your Githubs yesterday)

### CODE REVIEW

**Review by ajpaon — APPROVED** [review]

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,42 @@
...
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jennifer Dai's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
+      <script src="script.js"></script>
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/index.html]: Indented a little bit too much. Also, why did you move the script tag to the body? Not saying its  a bad thing. Just curious. While being curious, I found an [answer on stackoverflow](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup). You can keep it here but just an FYI on what actually happens when the browser looks at the HTML.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,42 @@
...
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <div id="header">
+        <h1>Jennifer Dai's Portfolio</h1>
+        <a href="https://ca.linkedin.com/in/jennifer-dai">LinkedIn</a>
+        <a href="https://github.com/nif33">Github</a>
+      </div>
+
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomGreeting()">Tell me more!</button>
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/index.html]: addRandomGreeting not descriptive on this new functionality. Maybe change it to addRandomFact.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,42 @@
...
       <div id="greeting-container"></div>
+
+      <h2>Background</h2>
+      <p>I'm a 2nd year Computing Science student at Simon Fraser University. Three fun facts about me: 
+          I have a pet dog named William, my favourite place I've travelled to is Montreal, 
+          and my first day of work at Google was also my 20th birthday!
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/index.html:24]: Happy late birthday!!!

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,42 @@
...
+      <img class="dog-slides" src="/images/dog/dog2.jpg">
+
+      <h2>My Favourite Places</h2>
+      <p>I haven't travelled much, but my favourite place I've been to is Montreal</p>
+      <img class="place-slides" src="/images/place/place0.jpg">
+      <img class="place-slides" src="/images/place/place1.jpg">
+      <img class="place-slides" src="/images/place/place2.jpg">
+      
     </div>
+  <script src="script.js"></script>
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/index.html:44]: Similar to my question above. This time why two scripts? I think you only need one. Please correct me if I'm wrong.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -17,12 +17,12 @@
  */
 function addRandomGreeting() {
   const greetings =
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/script.js]: Change this var to be more descriptive. Maybe "facts" or something similar.

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -17,12 +17,12 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I have a pet dog', 'I like to code'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/script.js]: Change var name here as well.

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -1,9 +1,75 @@
 #content {
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/style.css:1]: Overall, commits with UI/UX changes should have links to screenshots with how the website looks. Makes it better for code reviews and just easy to visualize what changed per commit. Could you add a screenshot link?

**Review by juanalonsocruz — CHANGES_REQUESTED** [review]
Left some comments. Please address them and send it back for re-review.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (nif33) [reply portfolio/src/main/webapp/index.html]: I meant to remove this line. I moved the script tag to the end of the body because I am currently working on changing the image display to a slideshow and the simplest way I could find to do it requires that the script be at the end to display the images.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (nif33) [reply portfolio/src/main/webapp/index.html]: Done

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (nif33) [reply portfolio/src/main/webapp/index.html:24]: Thanks!

File: portfolio/src/main/webapp/style.css (hunk 7 above)
Comment (nif33) [reply portfolio/src/main/webapp/style.css:1]: I do not have a screenshot link for before the changes, but here is what it currently looks like: [removed link because I used snipit]

Comment (ajpaon) [reply portfolio/src/main/webapp/style.css:1]: One of the most important reasons to use a tool like git is that your code history is always accessible.

This means you are able to check out an earlier version, get a screenshot from that version, and add it to this commit. Let us know if you need any help with that.

Also, we should find a screenshot solution that works outside Google as this is an open source project.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (juanalonsocruz) [reply portfolio/src/main/webapp/index.html:44]: Make sure to resolve comments after you fix them. Please do this for the other comments as well.

File: portfolio/src/main/webapp/script.js [hunk 8]
```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
+function addRandomFact() {
+  const facts =
       ['I have a pet dog', 'I like to code'];
 
   // Pick a random greeting.
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/script.js]: Nit: Also change this comment.

**Review by juanalonsocruz — APPROVED** [review]
